packages/grid/x-data-grid-pro/src/hooks/features/export/gridExcelExportInterface.ts

```diff
@@ -0,0 +1,61 @@
+import type * as Excel from 'exceljs';
+import {
+  GridFileExportOptions,
+  GridExportFormat as GridExportFormatCommunity,
+  GridExportExtension as GridExportExtensionCommunity,
+  GridToolbarExportProps as GridToolbarExportPropsCommunity,
+  GridExportDisplayOptions,
+} from '@mui/x-data-grid';
+
+export type GridExportFormat = GridExportFormatCommunity | 'excel';
+export type GridExportExtension = GridExportExtensionCommunity | 'xlsx';
+
+export interface GridExceljsProcessInput {
+  workbook: Excel.Workbook;
+  worksheet: Excel.Worksheet;
+}
+
+/**
+ * The options to apply on the Excel export.
+ */
+export interface GridExcelExportOptions extends GridFileExportOptions {
+  /**
+   * Name given to the worksheet containing the columns valueOptions.
+   * valueOptions are added to this worksheet if they are provided as an array.
+   */
+  valueOptionsSheetName?: string;
+  /**
+   * Method applied before adding the grid content to the workbook
+   * @param {GridExceljsProcessInput} processInput object containing the workbook and the worksheet
+   * @returns {Promise<void>} Modifications are done by mutating the workbook or the worksheet
+   * */
```

Comment:
```suggestion
   * Method called before adding the rows o the workbook.
   * @param {GridExceljsProcessInput} processInput object containing the workbook and the worksheet.
   * @returns {Promise<void>} A promise which resolves after processing the input.
   * */
```

Same for `exceljsPostProcess`.